Classes/Eel/IndexingHelper.php

```diff
@@ -50,7 +50,7 @@ public function buildAllPathPrefixes($path)
 
         $currentPath = '';
         $pathPrefixes = [];
-        if ($path{0} === '/') {
+        if (strpos($path, '/') === 0) {
```

Comment:
Why not just this?
```suggestion
        if ($path[0]) === '/') {
```

Reply:
Sorry, I just used the same code from the 4.0 and master branches without doing actual thinking. Should I adjust or is it better to have it in sync?

Reply:
Ah, if it's like that let's keep it in sync! Sorry, I didn't do any further research myself… 🙈